builds/cmake/ZeroMQConfig.cmake.in

```diff
@@ -19,7 +19,11 @@
 if(NOT TARGET libzmq AND NOT TARGET libzmq-static)
   include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
 
-  get_target_property(@PROJECT_NAME@_INCLUDE_DIR libzmq INTERFACE_INCLUDE_DIRECTORIES)
-  get_target_property(@PROJECT_NAME@_LIBRARY libzmq LOCATION)
-  get_target_property(@PROJECT_NAME@_STATIC_LIBRARY libzmq-static LOCATION)
+  if (TARGET libzmq)
+    get_target_property(@PROJECT_NAME@_INCLUDE_DIR libzmq INTERFACE_INCLUDE_DIRECTORIES)
+    get_target_property(@PROJECT_NAME@_LIBRARY libzmq LOCATION)
+  endif()
+  if (TARGET libzmq-static)
```

Comment:
Is @PROJECT_NAME@_INCLUDE_DIR something that should be set using the libzmq-static target if the shared libzmq target doesn't exist?